core/deps/libwebsocket/sha-1.c

```diff
@@ -99,8 +99,6 @@ static const unsigned int _K[] =
 		sha1_step(ctxt);		\
 	}
 
-static void sha1_step __P((struct sha1_ctxt *));
```

Comment:
This should be `static void sha1_step (struct sha1_ctxt *);`, as __P is part of cdefs. Seems to compile fine without the declaration though?